### PR TITLE
Rework dialogue to line 184, add choices

### PR DESCRIPTION
**Pull request by SamGloverWrites**

### CODE REVIEW

File: vn/game/script.rpy [hunk 1]
```diff
@@ -181,61 +134,44 @@ label start:
...
 
-    hide consequentialist
-    with Dissolve(.25)
+    label walk_forward: 
+        if menu_flag:
+            "From the sound I made earlier, I guessed I was in an open space, but no-"
 
-    show blindpriest at center:
+        else:
+            "Not for long, as I was stopped in my tracks by something tall, thin, metallic -
```
Comment (steeznson) [inline vn/game/script.rpy:150]: I think you need the `"""` syntax for multiline comments. Might be wrong about this so ignore me if it works,

**Review by steeznson — COMMENTED** [review]
LGTM - one query

File: vn/game/script.rpy [hunk 2]
```diff
@@ -60,35 +45,20 @@ label start:
...
-        even if you could see? So then, the whole thing falls apart.
+        'I can't see anything,' is what's meant.
+        Those who say, 'I can't see shit' instead say a lot about how they feel about anything; 
+        apparently, it's interchangeable with fecal matter. How miserable indeed.
     """
 
     show consequentialist:
-        xalign 0.85
-        yalign 0.70
+        xalign 0.85        yalign 0.70
```
Comment (ADover18) [inline vn/game/script.rpy:57]: Suggested Change to keep formatting:
```
        xalign 0.85
        yalign 0.70
 ```

File: vn/game/script.rpy (hunk 1 above)
Comment (SamGloverWrites) [reply vn/game/script.rpy:150]: Ah no worries, I'll amend and push it again

Comment (steeznson) [reply vn/game/script.rpy:150]: Yeah you can just add a new commit and push to the same branch. It should update the PR automagically

**Review by ADover18 — CHANGES_REQUESTED** [review]

**Comment by steeznson** [comment]
You didn't need to close the PR! There should be a button somewhere on here to re-open it. The branch still exists.

File: vn/game/script.rpy [hunk 3]
```diff
@@ -181,61 +134,44 @@ label start:
...
-    with Dissolve(.25)
+    label walk_forward: 
+        if menu_flag:
+            "From the sound I made earlier, I guessed I was in an open space, but no-"
 
-    show blindpriest at center:
+        else:
+            "Not for long, as I was stopped in my tracks by something tall, thin, metallic -
+            and legion."
+
```
Comment (ADover18) [inline vn/game/script.rpy:152]: Suggested Change:
```
hide blindpriest
    with Dissolve(.25)
```

**Pull request closed without merging**